index.js

```diff
@@ -419,6 +419,52 @@
     return f(y, x);
   });
 
+  //# lift :: Applicative f => (a -> b) -> f a -> f b
```

Comment:
Should this be `Apply` rather than `Applicative`? We don't need the full power of Applicative, do we?